SPEC.md

```diff
@@ -70,21 +70,19 @@ This set of formats makes it easy to build, host and secure a container using te
 - Image archives MUST be a tar formatted file with no duplicate entries.
 - Image archives MUST have only two top-level pathnames, `manifest` (a regular file) and `rootfs` (a directory). Image archives with additional files outside of `rootfs` are not valid.
 - All files in the image MUST maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
+- Image archives MAY be compressed with `gzip`, `bzip2`, or `xz`.
+- Image archives MAY be encrypted with AES symmetric encryption, after (optional) compression.
 - Image archives MUST be named with the suffix `.aci`, irrespective of compression/encryption (see below).
 - Image archives SHOULD be signed using PGP, the format MUST be ascii-armored detached signature mode.
```

Comment:
actually could you please add "after (optional) compression/encryption" to be completely explicit, as the list is not technically ordered